src/Files.App/Actions/Content/Install/InstallCertificateAction.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System.Security.Cryptography;
+using System.Security.Cryptography.X509Certificates;
+using Files.App.Commands;
+using Files.App.Contexts;
+using Files.Backend.Helpers;
+
+namespace Files.App.Actions;
+
+internal class InstallCertificateAction : ObservableObject, IAction
+{
+	private readonly IContentPageContext context = Ioc.Default.GetRequiredService<IContentPageContext>();
+
+	public string Label => "Install".GetLocalizedResource();
+
+	public string Description => "InstallCertificateDescription".GetLocalizedResource();
+
+	public RichGlyph Glyph { get; } = new("\uEB95");
+	
+	public bool IsExecutable =>context.SelectedItems.All(x => FileExtensionHelpers.IsCertificateFile(x.FileExtension)) &&
+	                            context.PageType is not ContentPageTypes.RecycleBin and not ContentPageTypes.ZipFolder;
+
+	public InstallCertificateAction()
+	{
+		context.PropertyChanged += Context_PropertyChanged;
+	}
+
+	public Task ExecuteAsync()
+	{
+		foreach (ListedItem selectedItem in context.SelectedItems)
+		{
+			try
+			{
+				X509Certificate2 certificate = new X509Certificate2(selectedItem.ItemPath);
+				X509Store store = new X509Store(StoreName.Root, StoreLocation.CurrentUser);
```

Comment:
I think that File Explorer lets the user choose the location, if possible, we should do the same. A good approach might be to use the context menu verb for "install".

Reply:
Yeah I looked into this, when you open a certificate and choose to install it first you need to decide the store location 
![image](https://github.com/files-community/Files/assets/88297220/92eaba6b-d6c5-4e16-abd1-9c095b3f51c0)
and then you can decide where you want to save it 
![image](https://github.com/files-community/Files/assets/88297220/7bbcf000-5638-4a34-8179-5f8fa1b7435f)
so I was thinking I could do a menu that lets you choose between current user and local machine and then have submenus for the location, but I wanted to know if you think that could work well, we could also only allow to install directly to the current user and not the machine

Reply:
>  when you open a certificate and choose to install it first you need to decide the store location

Can we open the same screen File Explorer does?

Reply:
I haven't found a way to do it

Reply:
it seems it is a process that we are not able to call

Reply:
@br1trs you can refer this code.
https://github.com/files-community/Files/blob/6d7c29d4037362602f2146cb0b9fab504999d18b/src/Files.App/Actions/Content/Run/RunAsAdminAction.cs#L29-L32

The verb seems to be "add".
![image](https://github.com/files-community/Files/assets/66369541/e5acb093-1031-4ced-bead-b3c3743fc8e1)

Reply:
it works!!!